src/components/dataTable/AssertedGenes.js

```diff
@@ -1,11 +1,11 @@
 import CollapsibleList from '../collapsibleList/collapsibleList';
-import ExternalLink from '../ExternalLink';
+import { Link } from 'react-router-dom';
 
 
 function makeAssertedGeneLink(curie, geneSymbol) {
     if(curie) {
       const symbol = <span dangerouslySetInnerHTML={{__html: geneSymbol}}/>;
-      return <ExternalLink href={`https://www.alliancegenome.org/gene/${curie}`}>{symbol}</ExternalLink>;
+      return <Link to={`/gene/${curie}`}>{symbol}</Link>;
```

Comment:
@adamgibs thanks this is weird that this was a absolute link

Reply:
Yeah, very strange.